sample/Pizzeria.Store.Data/StoreDbContext.cs

```diff
@@ -18,4 +18,6 @@ public StoreDbContext(DbContextOptions<StoreDbContext> options)
     public DbSet<Pizza> Pizzas { get; set; }
 
     public DbSet<Order> Orders { get; set; }
+
+    public DbSet<OrderQueryProjection> OrderQueryProjections { get; set; }

```

Comment:
I believe there is already a `QueryProjectionReferences` property on `Lewee.Infrastructure.Data.ApplicationDbContext`, which will be inherited by this class